### ISSUE TITLE
Address JPEG vs TIFF for raw file formats?

### ISSUE THREAD

**Issue by ewallace** [carpentries-lab/good-enough-practices#143]
The file formats part of data management episode is focused on .csv files and other text-based formats.

It might be helpful to say something about images and .JPEG vs .TIFF, and link to the blogpost from ASBMB/Kaoru Sakabe, combating compression https://www.asbmb.org/asbmb-today/science/050117/combating-compression

**Comment by alee** [comment]
Good point - is the key point to make sure to save and publish your data in lossless open formats? There can be a tradeoff there in file size and efficiency though as tiffs are much larger than basic jpegs. JPEG 2000 is a lossless file format that could stand in as a reasonable choice that's more efficient than tiffs for rendering / processing / storage, and PNG files are also lossless.

https://blogs.loc.gov/thesignal/2014/05/comparing-formats-for-still-image-digitizing-part-two/
